pages/library/_viewId.vue

```diff
@@ -9,7 +9,7 @@
     </v-row>
     <v-row>
       <v-col v-for="item in items" :key="item.Id" cols="6" sm="4" md="3" lg="2">
-        <card :item="item" />
+        <card :item="item" episode />
```

Comment:
Seems like you changed the wrong one.

This one should stay as the default, to show production year and such.

Reply:
Oops :p

Reply:
Should be fixed